src/components/common/dashboard/Dashboard.vue

```diff
@@ -0,0 +1,223 @@
+                    title="Finished Operations"
+                    description="Completed Operations. Marking them as read will remove them from the list."
+                    @marked-read="markRead"
+                />
+                <dashboard-component
+                    class="mt-5 lg:mt-0 lg:w-1/2"
+                    identifier="actionRequired"
+                    :enrollment-id="enrollmentId"
+                    :operations="shownActionRequiredOperations"
+                    title="Action Required"
+                    description="Operations that require your approval for completion."
+                />
+            </div>
+            <dashboard-component
+                identifier="pending"
+                :enrollment-id="enrollmentId"
+                :operations="pendingOwnOperations"
+                title="Pending Operations"
+                description="Your pending operations that wait for approval."
+            />
+            <button v-if="isAdmin" class="btn btn-blue-primary mt-8 p-2" @click="routeAllOperationsPage">Show all operations</button>
+            <div class="w-full flex items-start mt-10">
+                <label class="text-sm text-gray-700">
+                    Note: You can find a complete archive of your operations
+                    <router-link
+                        id="showOperationsArchive"
+                        :to="{ name: 'operations.archive' }"
+                        class="navigation-link hover:cursor-pointer hover:underline"
+                    >
+                        here
+                    </router-link>
+                    .
+                </label>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+    import { computed, onBeforeMount, ref } from "vue";
+    import LoadingSpinner from "@/components/common/loading/Spinner.vue";
+    import Operation, { ApprovalList } from "@/api/api_models/operation_management/Operation";
+    import { UC4Identifier } from "@/api/helpers/UC4Identifier";
+    import { OperationStatus } from "@/api/api_models/operation_management/OperationState";
+    import DashboardComponent from "@/components/common/dashboard/DashboardComponent.vue";
+    import { useStore } from "@/use/store/store";
+    import SearchBar from "@/components/common/SearchBar.vue";
+    import { MutationTypes } from "@/use/store/mutation-types";
+    import { Role } from "@/entities/Role";
+    import Router from "@/use/router";
+    import CertificateManagement from "@/api/CertificateManagement";
+    import GenericResponseHandler from "@/use/helpers/GenericResponseHandler";
+    import OperationManagement from "@/api/OperationManagement";
+    import filterOperations from "@/use/helpers/filterOperations";
+
+    export default {
+        name: "Dashboard",
+        components: {
+            LoadingSpinner,
+            DashboardComponent,
+            SearchBar,
+        },
+        setup() {
+            const busy = ref(0);
```

Comment:
That's because we have multiple points within the component, where we start multiple, but NOT the same async calls. With a number, we count how many async procedures are not finished yet. Using a boolean here would be difficult, as we would set the boolean to `false` at some point although another call is not finished